Material.Avalonia/Material.Avalonia.csproj

```diff
@@ -2,7 +2,7 @@
 
     <PropertyGroup>
         <TargetFramework>netstandard2.0</TargetFramework>
-        <VersionPrefix>2.5.0</VersionPrefix>
+        <VersionPrefix>3.0.0-rc0</VersionPrefix>
```

Comment:
Might facing issue when packing NuGet package for dev build

Reply:
@appleneko2001 version `3.0.0-rc0.100-nightly` is valid